Springer Link.js

```diff
@@ -196,6 +196,7 @@ function complementItem(doc, item) {
 			item.abstractNote = abs.trim();
 		}
 		item.abstractNote = titleTextGerman + "\n\n" + ZU.trimInternal(abs).replace(/^Abstract[:\s]*/, "");
+		if (item.abstractNote == null) item.abstractNote = '';
```

Comment:
"lastUpdated" in file header should be changed as well

Reply:
... and are you sure this works? I guess you should compare to the string `'null'` rather than the `null` value

Reply:
yes in zotkat it works
![image](https://user-images.githubusercontent.com/1545743/88777632-b7a42700-d187-11ea-8285-25cc55d78c7a.png)

Reply:
OK